core/src/main/java/org/apache/accumulo/core/client/mapred/AccumuloOutputFormat.java

```diff
@@ -552,13 +550,9 @@ public void close(Reporter reporter) throws IOException {
       } catch (MutationsRejectedException e) {
         if (!e.getSecurityErrorCodes().isEmpty()) {
           HashMap<String,Set<SecurityErrorCode>> tables = new HashMap<>();
-          for (Entry<TabletId,Set<SecurityErrorCode>> ke : e.getSecurityErrorCodes().entrySet()) {
+          for (var ke : e.getSecurityErrorCodes().entrySet()) {
             String tableId = ke.getKey().getTableId().toString();
-            Set<SecurityErrorCode> secCodes = tables.get(tableId);
-            if (secCodes == null) {
-              secCodes = new HashSet<>();
-              tables.put(tableId, secCodes);
-            }
+            Set<SecurityErrorCode> secCodes = tables.computeIfAbsent(tableId, k -> new HashSet<>());
```

Comment:
One of the great things about `.computeIfAbsent()` is that you don't need to assign it to a variable... you could just immediately use it on the next line:

```java
tables.computeIfAbsent(tableId, k -> new HashSet<>()).addAll(ke.getValue());
```